app/assets/stylesheets/training_modules/_cards.styl

```diff
@@ -17,6 +17,8 @@
   padding 10px 11px
   width 50px
   height 50px
+  position absolute
+  right 0px 
```

Comment:
Do these changes have any effect on the non-mobile layout?

Reply:
I tested it and see that they cancel each other out; 'absolute' introduces space to the right of the box holding the arrow icon, but `right 0` removes it again. LGTM